Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -281,6 +281,7 @@ export default function widgetBehavior(publicAPI, model) {
       ) {
         if (model.activeState.setOrigin) {
           model.activeState.setOrigin(worldCoords);
+          publicAPI.setMoveHandleVisibility(!publicAPI.isPlaced());
```

Comment:
it does not make sense to call setMoveHandleVisibility if the activeState is not a moveHandle.

I suggest instead to do: `updateHandleVisibility(getHandleIndex(model.activeState));`

(I'm not sure of the "render()" call in `updateHandleVisiblity()` though...

Reply:
I'm not sure to understand...

`updateHandleVisibility` does not update the `visible` mixin, so we would still have to use 
```js
moveHandle.setVisible(!publicAPI.isPlaced())
``` 
for this to work.

Also, the `getHandleIndex` function does not exists and I'm not sure how to compute the index in a smart way (but I can still do it with a `switch`)

Wouldn't it be easier to add a condition like this?
```js
if (model.activeState === model.widgetState.getMoveHandle()) {
    publicAPI.setMoveHandleVisibility(!publicAPI.isPlaced());
}
```